src/components/Sidebar.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./Sidebar.css";
+import Home from "../components/icons/home";
+import Menu from "../components/icons/menu";
+import Down from "../components/icons/down";
+
+interface SidebarProps {
+  setTitle: (title: string) => void;
+}
+
+const Sidebar: React.FC<SidebarProps> = ({ setTitle }) => {
+  const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+  return (
+    <div className="sidebar">
+      <div className="menu-item" onClick={() => setTitle("Start")}>
+        <Home /> Start
+      </div>
+      <div className="menu-item" onClick={() => setShowDropdown(!showDropdown)}>
+                  <Menu />Machine Learning &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<Down/>
```

Comment:
Icon for expanding/collapsing should change between up and down, to reflect menu open or menu closed. Check example here: 
![image](https://github.com/user-attachments/assets/b461b32e-d5a1-4c03-8d91-88bc443de56d)